src-ghc/Pact/Interpreter.hs

```diff
@@ -61,14 +61,19 @@ data EvalResult = EvalResult
 
 
 evalExec :: EvalEnv e -> ParsedCode -> IO EvalResult
-evalExec evalEnv ParsedCode {..} = do
+evalExec env pc = evalExecState def env pc
```

Comment:
Hate to ask this but can you add explicit exports? Everything except `evalTerms` would be exported.

Reply:
Sure - I'll amend